include/evmc/evmc.hpp

```diff
@@ -114,18 +114,18 @@ using uint256be = bytes32;
 
 
 /// Loads 64 bits / 8 bytes of data from the given @p bytes array in big-endian order.
-constexpr inline uint64_t load64be(const uint8_t* bytes) noexcept
+constexpr inline uint64_t load64be(const uint8_t* bs) noexcept
```

Comment:
Why not `data`? Also the comment still says `bytes`.

Reply:
Changed to `data`.